hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/util/WebAppUtils.java

```diff
@@ -509,11 +512,18 @@ static String getPassword(Configuration conf, String alias) {
       char[] passchars = conf.getPassword(alias);
       if (passchars != null) {
         password = new String(passchars);
+        LOG.info("Successful password retrieval for alias: {}", alias);
```

Comment:
I think debug level will be enough here, to avoid too much log record.

Reply:
I am not so sure, because when I try with DEBUG level and run the unit test the log message is not shown. I think it is override by other log level that is why I used INFO level to see it every time.

Is there a way to see a DEBUG level logs in some cases when the user wants?

Reply:
Yup, seems you are right